tool/automatiek/vendor.rb

```diff
@@ -74,6 +74,26 @@ def relative_require_target_from(file, entrypoint = require_entrypoint)
   def require_target
     @require_target ||= vendor_lib.sub(%r{^(.+?/)?lib/}, "") << "/lib"
   end
+
+  def cdx_component(spec)
```

Comment:
This file seems to have too much responsibility currently. Any chance to split a little?